src/components/dashboard/SendLinkSummary.js

```diff
@@ -155,6 +156,14 @@ const SendLinkSummary = ({ screenProps, styles }: AmountProps) => {
           userStorage.markWithErrorEvent(txHash)
         },
       })
+      const { txPromise } = generateLinkResponse
+
+      txPromise.catch(e => {
+        log.error(e.message, e)
+        showErrorDialog('Something unexpected happened', '', {
```

Comment:
error should be about sending money failed, look for other error messages and copy.
boldMessage, same, look for other examples

Reply:
Set a message `Could not complete transaction. Please try again` as in the code below